eval.py

```diff
@@ -396,8 +398,7 @@ def get_occluded_stats( num_tp_occluded: Number, num_gt_occluded: Number,
         if (num_gt_occluded > 0) else 0.0
     fscore = hmean(recall, precision)
 
-    stats = {'occluded_recall': recall,
-             'occluded_fscore': fscore }
+    stats = {prefix + "_recall": recall, prefix + "_fscore": fscore}
```

Comment:
The underscore shouldn't be part of the default key. If no prefix is used (there was was a use case; I haven't followed whether it still exists in the current form), we wouldn't want a naked leading underscore. 

Removing the underscore here (as proposed) would mean needing to add it to the calls below.

Another way to proceed (arguably less elegant), would be to check for an empty prefix and only insert the underscore here when prefix is non-empty.